### PR TITLE
Add Metrology

### PR DESCRIPTION
**Pull request by schuetzepaul**
This adds a light-weight metrology for fully assembled modules.

There's a new tab in the main window called "Metrology". The usual settings are available there, similar to the alignment routine. The metrology needs to be started at the position of the top-left PSp fiducial marker, which should be located around the "RefMarker" position. I suggest to furthermore enable the vacuum on the baseplate. Hence, I added two buttons to a) move to the initial position and b) to enable the vacuum on the baseplate.

Then the SW performs pattern recognitions on the four top markers and calculates the following quantities:

- dx/dy: (position_PSs_top_left_marker - position_PSp_top_left_marker) in both x and y
- dx/dy corrected: the upper value, corrected for the target (configuration parameter) offset between the two markers
- PSp/PSs angle: angle of PSp and PSs in the stage coordinate system
- Angle between PSp and PSs: the difference between the above

One could consider using all four markers for better statistics, but this would require to also create and load templates for the bottom markers. For a first check of the alignment, the part implemented here should be ok.

### CODE REVIEW

File: assembly/assembly/parameters/SiDummyPS.cfg
Comment (anurnberg) [inline assembly/assembly/parameters/SiDummyPS.cfg:1]: Did we change the reference marker on the platform?

File: assembly/assembly/parameters/glass.cfg [hunk 1]
```diff
@@ -82,8 +82,8 @@ FromPSSPlusSpacersToMaPSAPositionToGluingStage_dY       -5.0
 
 # distance: from ref-point on assembly platform to spacer's edge corresponding to sensor ref. marker (in spacers platform orientation -- now same as baseplate orientation)
```
Comment (anurnberg) [inline assembly/assembly/parameters/glass.cfg:83]: Should this be also added for the Silicon file?

**Comment by schuetzepaul** [comment]
Ready for review (again) :-)

**Comment by schuetzepaul** [comment]
Will merge this PR, as all comments were related to the parameter changes, which are now removed.

**Pull request merged**